src/carbon-esm-styleguide.js

```diff
@@ -0,0 +1,3 @@
+import "../node_modules/carbon-components/css/carbon-components.min.css";
```

Comment:
I guess a better variant uses the index.html of the app shell - we should `<link>` the stylesheets instead imho.